pkg/apis/serving/k8s_validation.go

```diff
@@ -267,7 +276,51 @@ func ValidatePodSpec(ps corev1.PodSpec) *apis.FieldError {
 	return errs
 }
 
+//ValidateMultiContainerPorts validates port when specified multiple containers
+func ValidateMultiContainerPorts(containers []corev1.Container) *apis.FieldError {
+	cPort := []int32{}
+	var errs *apis.FieldError
+	for i := range containers {
+		for j := range containers[i].Ports {
+			cPort = append(cPort, containers[i].Ports[j].ContainerPort)
```

Comment:
This seems to be doing two checks here:
* Multiple ports on a single container
* Multiple containers with a ports specified

However, `validateContainerPorts` already validates if multiple ports are specified per container and does so with a different error message.

Reply:
Handled port validation for both single and multi container

Thank you